assembl/tests/fixtures/graphql.py

```diff
@@ -28,12 +30,97 @@ def graphql_participant1_request(request, test_participant1_webrequest, discussi
+        image_url = u'https://example.net/image.jpg',
+        is_thematics_table = True
+    )
+
+    thread = DiscussionPhase(
+        discussion = discussion,
+        identifier = 'thread',
+        title = LangString.create(u"thread phase title fixture", "en"),
+        description = LangString.create(u"thread phase description fixture", "en"),
+        start = datetime(2018, 2, 16, 9, 0, 0),
+        end = datetime(2018, 3, 15, 9, 0, 0),
+        interface_v1 = False,
+        image_url = u'https://example.net/image.jpg'
+    )
+
+    multiColumns = DiscussionPhase(
+        discussion = discussion,
+        identifier = 'multiColumns',
+        title = LangString.create(u"multiColumns phase title fixture", "en"),
+        description = LangString.create(u"multiColumns phase description fixture", "en"),
+        start = datetime(2018, 3, 16, 9, 0, 0),
+        end = datetime(2018, 4, 15, 9, 0, 0),
+        interface_v1 = False,
+        image_url = u'https://example.net/image.jpg'
+    )
+
+    voteSession = DiscussionPhase(
+        discussion = discussion,
+        identifier = 'voteSession',
+        title = LangString.create(u"voteSession phase title fixture", "en"),
+        description = LangString.create(u"voteSession phase description fixture", "en"),
+        start = datetime(2018, 4, 16, 9, 0, 0),
+        end = datetime(2018, 5, 15, 9, 0, 0),
+        interface_v1 = False,
+        image_url = u'https://example.net/image.jpg',
+        is_thematics_table = True
+    )
+
+    brightMirror = DiscussionPhase(
+        discussion = discussion,
+        identifier = 'brightMirror',
+        title = LangString.create(u"brightMirror phase title fixture", "en"),
+        description = LangString.create(u"brightMirror phase description fixture", "en"),
+        start = datetime(2018, 6, 16, 9, 0, 0),
+        end = datetime(2018, 7, 15, 9, 0, 0),
+        interface_v1 = False,
+        image_url = u'https://example.net/image.jpg',
+        is_thematics_table = True
+    )
+
+    # Create the phase
+    test_session.add(survey)
+    test_session.add(thread)
+    test_session.add(multiColumns)
+    test_session.add(voteSession)
+    test_session.add(brightMirror)
+    test_session.flush()
+
+    def fin():
+        print "finalizer timeline"
+        test_session.delete(survey)
+        test_session.delete(thread)
+        test_session.delete(multiColumns)
+        test_session.delete(voteSession)
+        test_session.delete(brightMirror)
+        test_session.flush()
+
+    request.addfinalizer(fin)
+    phases = test_session.query(models.DiscussionPhase).all()
+    return {p.identifier: p for p in phases}
+
+
+@pytest.fixture(scope="function")
+def idea_in_thread_phase(phases, graphql_request):
     from assembl.graphql.schema import Schema as schema
     res = schema.execute(u"""
 mutation myFirstMutation {
     createThematic(
-        identifier: "thread",
+        discussionPhaseId: """+unicode(phases['thread'].id)+u""",
```

Comment:
Is this a pythonic style, or literally `+unicode(phases['thread'].id)+u` ?

Reply:
Not the best way to write it, but it was far more easier to update the tests like this. It was time consuming enough to adapt all the tests.